openssl-dynamic/src/main/java/io/netty/tcnative/jni/ocsp/OcspCallback.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.tcnative.jni.ocsp;
```

Comment:
Sooo, on my end we don't need to support "old" versions and we'll use whatever latest SNAPSHOT of Netty is using.